### ISSUE TITLE
[1.20.1 v0.6.8 Hotfix] Is the power system wip?

### ISSUE THREAD

**Issue by C4PTRAMPAGE** [Povstalec/StargateJourney#41]
Overall the stargate doesn't seem to respect power config settings for connection and maintenance.

**Comment by Povstalec** [comment]
Yes, the power system on the versions you all use is still WIP. One of the reasons it's disabled by default.

**Comment by C4PTRAMPAGE** [comment]
alright, thanks.
